src/utils/invariant.js

```diff
@@ -24,7 +24,7 @@
 
 var validateFormat = function(format) {};
 
-if (__DEV__) {
+if (global.__DEV__) {
```

Comment:
Any negative ramifications for this? It was causing a `ReferenceError` when testing the production web build.

Reply:
It's probably fine -- but one other way to solve it would be to polyfill __DEV__ in webpack:

```
new webpack.DefinePlugin({
      __DEV__: isDebug,
    }),
```

Reply:
Seems that's how it's [already being done](https://github.com/react-community/react-navigation/blob/master/website/config/webpack.config.prod.js#L206). Not sure why it isn't working 🤔